subprocess.h

```diff
@@ -306,63 +313,6 @@ struct subprocess_overlapped_s {
   void *hEvent;
 };
 
-#pragma warning(pop)
```

Comment:
This was all done for a very specific reason - namely to make compile time fast, no way I'll accept removing these!